.circleci/config.yml

```diff
@@ -0,0 +1,42 @@
+version: 2.0
+jobs:
+  build:
+    # CircleCI does not run workflows for tags unless you explicitly specify tag filters
+    filters:
+      tags:
+        only:
+          - /^(.*)$/
+
+    docker:
+      - image: circleci/openjdk:11-jdk
+
+    steps:
+      - checkout
+
+      # https://circleci.com/docs/2.0/building-docker-images/
+      - setup_remote_docker:
+          docker_layer_caching: true
+
+      - restore_cache:
+          keys:
+            - sbt-cache-{{ checksum "app-backend/project/Dependencies.scala" }}
+
+      - run:
+          command: sbt test:compile
```

Comment:
Split out fetching dependencies from the rest of the build for more efficient caching.